### PR TITLE
DAOS-15999 test: Update led check method for soak_utils.py

### PR DESCRIPTION
**Pull request by rpadma2**
Summary: led identify commands are sent to the appropriate host which has the valid UUIDs

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: src/tests/ftest/util/soak_utils.py [hunk 1]
```diff
@@ -429,19 +430,26 @@ def launch_vmd_identify_check(self, name, results, args):
...
+                dmg.storage_led_identify(ids=uuid, timeout=2)
+                # check if led is blinking
+                result = dmg.storage_led_check(ids=uuid)
+                # determine if leds are blinking as expected
+                for value in list(result['response']['host_storage_map'].values()):
+                    if value['storage']['smd_info']['devices']:
+                        for device in value['storage']['smd_info']['devices']:
+                            if device['ctrlr']['led_state'] != "QUICK_BLINK":
+                                failing_vmd.append([device['ctrlr']['pci_addr'], value['hosts']])
+                                status = False
```
Comment (daltonbohning) [inline src/tests/ftest/util/soak_utils.py:452]: `status` is never defined above so I expect `"status": status,` below could raise an exception.

File: src/tests/ftest/util/soak_utils.py [hunk 2]
```diff
@@ -429,19 +430,26 @@ def launch_vmd_identify_check(self, name, results, args):
     else:
         uuids = uuid_list
     self.log.info("VMD device UUIDs: %s", uuids)
+    host_uuids = get_storage_query_device_uuids(self.dmg_command)
+    for host, uuid_dict in host_uuids.items():
+        uuid_list = sorted(uuid_dict.keys())
+        self.log.info("Devices on hosts %s: %s", host, uuid_list)
```
Comment (daltonbohning) [inline src/tests/ftest/util/soak_utils.py:437]: host or hosts?

File: src/tests/ftest/util/soak_utils.py (hunk 1 above)
Comment (rpadma2) [reply src/tests/ftest/util/soak_utils.py:452]: status is defined on the latest update.
`    # pylint: disable=too-many-nested-blocks
    status = True

I accidentally removed it from the original code.

File: src/tests/ftest/util/soak_utils.py (hunk 2 above)
Comment (rpadma2) [reply src/tests/ftest/util/soak_utils.py:437]: It's just host... Need to fix the self.log.info message.

**Review by mjean308 — COMMENTED** [review]
It is working on aurora with 256 nodes;   I will bump it to 1024

**Review by daltonbohning — APPROVED** [review]

**Review by mjean308 — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Pull request merged**